src/qtt/algorithms/random_telegraph_signal.py

```diff
@@ -114,6 +114,19 @@ def plot_two_level_threshold(results, fig=100):
     plt.title('Result of two level threshold processing')
 
 
+def _create_integer_histogram(durations, verbose=0):
+    """ Calculate number of bins, bin edges and histogram for durations
+
+    This method works if the data is sampled at integer durations.
```

Comment:
When durations is empty array then error. When one duration is zero you get a negative bin_edge (-0.5). Is that oke?